### PR TITLE
[ANDROAPP-5434] Update libraries

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -189,8 +189,8 @@ android {
     configurations.all {
         resolutionStrategy {
             preferProjectModules()
-            force("junit:junit:4.12", "com.squareup.okhttp3:okhttp:3.12.0")
-            setForcedModules("com.squareup.okhttp3:okhttp:3.12.0")
+            /*force("junit:junit:4.12", "com.squareup.okhttp3:okhttp:3.12.0")
+            setForcedModules("com.squareup.okhttp3:okhttp:3.12.0")*/
```
Comment (ferdyrod) [inline app/build.gradle.kts:193]: Is there a reason this is commented?

File: app/src/main/java/org/dhis2/utils/analytics/AnalyticsInterceptor.kt [hunk 2]
```diff
@@ -40,15 +40,15 @@ class AnalyticsInterceptor(private val analyticHelper: AnalyticsHelper) : Interc
         val request = chain.request()
         val response = chain.proceed(request)
 
-        if (response.code() >= 400 && !isLogged) {
+        if (response.code >= 400 && !isLogged) {
             isLogged = D2Manager.getD2().userModule().blockingIsLogged()
         }
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/utils/analytics/AnalyticsInterceptor.kt:45]: I think this was changed in another PR, could cause conflicts

Comment (Balcan) [reply app/src/main/java/org/dhis2/utils/analytics/AnalyticsInterceptor.kt:45]: it is the same... it's just code is a variable not a method

File: app/build.gradle.kts (hunk 1 above)
Comment (Balcan) [reply app/build.gradle.kts:193]: true should be removed

**Review by andresmr — APPROVED** [review]

**Review by ferdyrod — APPROVED** [review]

**Pull request merged**